### PR TITLE
Fix month navigation

### PR DESCRIPTION
**Pull request by ricardoferrolho**

### CODE REVIEW

**Review by ziejka — DISMISSED** [review]
🚀

**Review by mnavarroespinosa — COMMENTED** [review]
I still see the issue when trying to navigate to the previous month

https://github.com/reedsy/vuejs-datepicker/assets/77450928/fd82e48b-d20c-4139-b573-4fb823aaa367

File: test/unit/specs/PickerDay/changeMonths.spec.js [hunk 1]
```diff
@@ -28,16 +28,16 @@ describe('PickerDay: changing months', () => {
...
-  it('changes the focused to the next month', () => {
+  it('changes the focused to the next month and resets the day', () => {
     wrapper.vm.nextMonth();
     expect(wrapper.emitted('update:focusedDate')).toBeTruthy();
-    expect(wrapper.emitted('update:focusedDate')[0]).toEqual([ new Date(2018, 2, 24).getTime() ]);
+    expect(wrapper.emitted('update:focusedDate')[0]).toEqual([ new Date(2018, 2, 1).getTime() ]);
   });
 
-  it('changes the focused to the next month', () => {
+  it('changes the focused to the next month and resets the day', () => {
```
Comment (mnavarroespinosa) [inline test/unit/specs/PickerDay/changeMonths.spec.js]:
```suggestion
  it('changes the focused to the previous month and resets the day', () => {
```

**Review by mnavarroespinosa — APPROVED** [review]
👌

**Pull request merged**